src/common/filefn.cpp

```diff
@@ -1185,15 +1185,14 @@ wxRenameFile(const wxString& file1, const wxString& file2, bool overwrite)
     // For explanation, see:  (warning...based mostly on observed behavior)
     //   http://bugzilla.audacityteam.org/show_bug.cgi?id=1266
     //   https://github.com/audacity/audacity/pull/94
-  unsigned long doserrno = 0;
+  errno = 0;
```

Comment:
This does not seems possible to me, `errno` is a macro, not a variable.
The code should be probably changed to:
`_set_errno(0)
`
See:
https://docs.microsoft.com/en-us/cpp/c-runtime-library/errno-doserrno-sys-errlist-and-sys-nerr?view=vs-2019